.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,99 @@
+          required: true
+        - label: This report is **not a support request or question**, both of which are better suited for either the [discussions section](https://github.com/qbittorrent/qBittorrent/discussions), [forum](https://qbforums.shiki.hu/), or [subreddit](https://www.reddit.com/r/qBittorrent/). The [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain a suitable solution either.
+          required: true
+        - label: I have taken note of the **(ADVISORY)** above & **added the settings file and relevant log(s)** in the **Logs & Settings/Config files** section at the bottom of the report.
+          required: true
+
+  - type: input
+    attributes:
+      label: Operating System & build/version.
+      description: Please specify if you are using 32-Bit or 64-Bit.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Qt & libtorrent-rasterbar versions.
+      description: (Linux OS Only)
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: qBittorrent version.
+      description: Please specify if you are using 32-Bit or 64-Bit.
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Browser name and version.
+      description: (If WebUI related Only)
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: What issue are you experiencing?
+      description: Please add "Crash Report" (if encountered) or give a clear and concise description of the bug/issue.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Steps to Reproduce.
+      description: Please walk us through it with a clear and concise description.
+      placeholder: |
+        1. [First Step]
+        2. [Second Step]
+        3. [and so on...]
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: What is the expected behavior?
+      description: Please give a clear and concise description of what you expected to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Additional context.
+      description: Add screenshots etc. (Anything that will give us more context about the issue you are encountering!)
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Log & Settings file(s).
+      description: Please add the settings file and at least the most recent log. See [this wiki page](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings) if you're not sure where to find them.
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+     value: |
+             ### Thanks for taking the time to fill out this report! (qBittorrent Bug Handler Team)
```

Comment:
I don't know if this will be visible in the posted issue. If it is, i would remove it completely but if it isn't (and you insist on keeping it) i would make it more formal: 
"Thank you for taking the time to fill out this report."

Reply:
> I don't know if this will be visible in the posted issue

No, it is not, not the first one at the least.
The `###` is superfluous.

> "Thank you for taking the time to fill out this report."

:+1:

Reply:
It's not visible & I don't insist on keeping any of it - just added it in as other repos use similar.
I will remove.

Reply:
it was removed.